.github/workflows/size-impact.yml

```diff
@@ -17,35 +14,14 @@ jobs:
       CI: true
     steps:
       - name: Setup git
-        env:
-          GITHUB_REPOSITORY_URL: ${{ github.repositoryUrl }}
-        run: |
-          git init
-          git remote add origin $GITHUB_REPOSITORY_URL
+        uses: actions/checkout@v2
       - name: Setup node ${{ matrix.node }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node }}
-      - name: Git checkout base
-        run: |
-          git fetch --no-tags --prune --depth=1 origin $GITHUB_BASE_REF
-          git checkout origin/$GITHUB_BASE_REF
-          git log -1
-      - name: Generate snapshot for base
-        run: |
-          npm install
-          npm run build
-          node ./.github/workflows/size-impact/generate-size-snapshot.js ../snapshot.base.json || echo "{}" > ../snapshot.base.json
-      - name: Git merge head
-        run: |
-          git fetch --no-tags --prune origin $GITHUB_REF
-          git merge FETCH_HEAD
-      - name: Generate snapshot after merge
-        run: |
-          npm install
-          npm run build
-          node ./.github/workflows/size-impact/generate-size-snapshot.js ../snapshot.head.json
-      - name: Report size impact
-        run: node ./.github/workflows/size-impact/report-size-impact.js ../snapshot.base.json ../snapshot.head.json
+      - name: Setup npm
+        run: npm install
+      - name: Report file size impact
+        run: node ./.github/workflows/size-impact/report-size-impact.js
```

Comment:
As you can see the workflow is not doing `git checkout` and `git merge` anymore.

It liked the expliciteness of seeing the commands but by doing it internally @jsenv/file-size-impact can catch errors and display them in the pull request message. It also makes the workflow file way shorter.